### PR TITLE
Add multiple select support

### PR DESCRIPTION
**Pull request by alanpoulain**

### CODE REVIEW

File: src/keep-current-value.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
     for (var i = 0; i < options.length; i++) {
       var el = angular.element(options[i].outerHTML);
-      if ( el.attr('selected') ) return el.attr('value');
-    };
-    if ( options[0] ) return angular.element(options[0]).attr('value');
+      if (el.attr('selected')) {
+        values.push(el.attr('value'));
+      }
+    }
+    if (element.attr('multiple') === 'multiple') {
```
Comment (bunzli) [inline src/keep-current-value.js]: I think you just need to check the presence the "multiple" attribute

`multiple="true"` and `multiple` should also work

**Comment by bunzli** [comment]
Thanks Alan! This looks great!.

Please don't include files from the build process. As soon as we merge this PR we will release a new version.

**Comment by alanpoulain** [comment]
Thanks! I have taken into account your comment and also removed the files from the build process.

**Comment by bunzli** [comment]
This looks awesome! Please squash the commits. I think with one commit will be ready to be merged. Thanks a lot!

**Comment by blackjid** [comment]
Maybe the test should be updated with the change of the multiple attribute, now is only testing with the attribute `multiple` with a value but not without.

**Comment by alanpoulain** [comment]
I think it is not necessary. The tests are done to test the Angular logic, not the jQuery one.

**Comment by blackjid** [comment]
I think you're right... jquery already takes care of it.

nevertheless, maybe we should use   `element.prop('multiple')` instead of `attr`.
The multiple attribute is a boolean attribute according to the [WC3 specification](http://www.w3.org/TR/html401/interact/forms.html#h-17.4). The [prop](http://api.jquery.com/prop/) method will return `true|false` on boolean methods regardless the value of the attribute.

what do you think?

**Comment by alanpoulain** [comment]
Even if multiple as a boolean is never used, you are right and we should stick to the W3C specification.
I have made the change.

**Comment by blackjid** [comment]
:+1:

**Comment by bunzli** [comment]
@alanpoulain merged and included in the new release 0.1.3

thanks!

**Pull request merged**